src/qt/bitcoingui.cpp

```diff
@@ -431,7 +431,10 @@ void BitcoinGUI::createActions()
 
             bool wallet_name_ok;
             //: Title of the Restore Wallet input dialog (where the wallet name is entered)
-            QString wallet_name = QInputDialog::getText(this, tr("Restore Name"), tr("Wallet Name:"), QLineEdit::Normal, "", &wallet_name_ok);
+            QString title = tr("Restore Wallet");
```

Comment:
you're changing the window title name from `Restore Name` to `Restore Wallet`. If this is intended, please document in the PR description why you are doing this. The mentioned review comment doesn't suggest this.

Reply:
Done.